src/utils/stringHashCode.ts

```diff
@@ -44,6 +44,7 @@ export function stringHashCode(value) {
 
     k1 = 0;
 
+    /*  eslint-disable no-fallthrough */
```

Comment:
This code is pretty dependent on the callthrough behaviour and is pretty tricky to understand, I'd prefer to leave it as is. What do you think?